.github/workflows/dev.yaml

```diff
@@ -54,26 +56,52 @@ jobs:
           registry: docker.stackable.tech
           username: github
           password: ${{ secrets.NEXUS_PASSWORD }}
+      - name: Login to Stackable Harbor
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # tag=v3
+        with:
+          registry: oci.stackable.tech
+          username: robot$sdp+github-action-build
+          password: ${{ secrets.HARBOR_ROBOT_SDP_GITHUB_ACTION_BUILD_SECRET }}
       - name: Publish dev images
         id: publish_images
         run: |
-          # bake images and load them to local docker repo for signing
+          # Bake images and load them to local docker repo for signing
           # TODO: buildx cannot --load and --push at the same time
           bake --product "${{ matrix.product }}" \
           --image-version 0.0.0-dev \
           --shard-count "${{matrix.shard_count}}" \
           --shard-index "${{matrix.shard_index}}" \
           --export-tags-file bake-target-tags
-          # push images to image repository
-          if [ -f bake-target-tags ]; then
-            docker image push "$(< bake-target-tags)"
-          fi
-      - name: Set up Cosign
-        uses: sigstore/cosign-installer@v3.1.2
-      - name: Sign images
-        run: |
+          # Push images to image repository
           if [ -f bake-target-tags ]; then
-            cosign sign -y "$(< bake-target-tags)"
+            echo "bake-target-tags: "$(< bake-target-tags)
+            IMAGE_NAME=$(cat bake-target-tags | cut -d ":" -f 1)
+            TAG_NAME=$(cat bake-target-tags | cut -d ":" -f 2)
+            echo "image: $IMAGE_NAME"
+            echo "tag: $TAG_NAME"
+            # Store the output of `docker image push` into a variable, so we can parse it for the digest
+            PUSH_OUTPUT=$(docker image push "$(< bake-target-tags)" 2>&1)
+            echo "$PUSH_OUTPUT"
+            # Obtain the digest of the pushed image from the output of `docker image push`, because signing by tag is deprecated and will be removed from cosign in the future
+            DIGEST=$(echo "$PUSH_OUTPUT" | awk "/: digest: sha256:[a-f0-9]{64} size: [0-9]+$/ { print \$3 }")
+            # Refer to image via its digest (docker.stackable.tech/stackable/airflow@sha256:0a1b2c...)
```

Comment:
This comment mentions stackable instead of sdp

```diff
- # Refer to image via its digest (docker.stackable.tech/stackable/airflow@sha256:0a1b2c...)
- # Refer to image via its digest (docker.stackable.tech/sdp/airflow@sha256:0a1b2c...)
```